src/GlobalStyles.js

```diff
@@ -0,0 +1,116 @@
+import styled, { createGlobalStyle } from 'styled-components';
+
+const GlobalStyles = createGlobalStyle`
+*, 
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+`
+export const SectionContainer = styled.section`
+  background-color: white;
+  display: flex; 
+  flex-direction: column;
+  justify-content: center;
+  padding: 5% 15%
+`
+
+export const SectionContainerGray = styled.section`
+  background-color: #f4f4f4;
+  display: flex; 
+  flex-direction: column;
+  justify-content: center;
+  padding: 5% 15%
+`
```

Comment:
If you'd like to copy an exisiting styled component to a new with just a small change like the background-color, you could write like this instead.  Then every css styling is the same except the css properties you write in the 'copy'. To have a little less code :)
```
export const SectionContainerGray = styled(SectionContainer)`
background-color: #f4f4f4
` 
```